app/assets/javascripts/osm.js.erb

```diff
@@ -63,12 +63,28 @@ OSM = {
 
   mapParams: function (search) {
     const params = OSM.params(search),
+          geoURI = URL.canParse(params.geouri) ? new URL(params.geouri) : { pathname: "", searchParams: [] },
           mapParams = {};
 
+    [geoURI.coordstring, ...geoURI.params] = geoURI.pathname.split(";");
+    geoURI.params = Object.fromEntries([...geoURI.searchParams, ...geoURI.params.map(p => p.split("=").map(decodeURIComponent))]);
+    geoURI.coords = L.latLng(geoURI.coordstring.split(","));
+    geoURI.epsg = geoURI.params.crs?.toUpperCase()?.replace(":", "");
+    if (geoURI.epsg?.startsWith("EPSG")) {
+      geoURI.coords = L.CRS.EPSG3857.unproject(L.CRS[geoURI.epsg]?.project(geoURI.coords));
+    }
```

Comment:
Is it just an excuse to parse all geo parameters or something that's actually supposed to work?

Reply:
`crs` can't even be `EPSG:something` according to the current rfc https://github.com/openstreetmap/openstreetmap-website/pull/5736#discussion_r2001173709